src/components/TurnInfoBoard.tsx

```diff
@@ -1,7 +1,23 @@
 import React from 'react';
+import TurnInfoCard, { ResultDto } from './TurnInfoCard';
 
-const TurnInfoBoard = () => {
-  return <div>모든 회차 정보 섹션 입니다.</div>;
+interface TurnInfoBoardProps {
+  results: Array<ResultDto | null>;
+  round: number;
+}
+
+const TurnInfoBoard = ({ results, round }: TurnInfoBoardProps) => {
+  const renderList = new Array<'Ready' | 'Playing' | 'Finished'>(results.length).fill('Finished');
+  if (round > results.length) renderList.push('Playing');
+  if (round > results.length + 1) renderList.push(...new Array(round - results.length - 1).fill('Ready'));
```

Comment:
제가 지식이 얕아서 괜한 걱정일 수도 있지만, (동일한 렌더링 내에 수행되는 작업인 것 같아 괜찮을 수도 있지만) 아무래도 배열을 변경하는 것이다 보니 순수성 보장에 대한 부분이 조금 우려가 되고, push되는 작업이 나눠서?처리되다 보니 비동기 관련해서도 사이드이펙트가 발생할 가능성 있지 않을까하는 부분도 염려가 되어서 처음 배열을 생성할 때 해당 상태 정보를 한 번에 처리하는 건 어떤지 제안드립니다!

```suggestion
  const renderList = Array.from({ length: round }, (v, i) => {
    if (i === results.length) return 'Playing';
    if (i > results.length) return 'Ready';
    return 'Finished';
  });
```
`from` 함수 사용한 것은 제가 지금 생각나는 방식이 이것 밖에 없어서 `from` 사용한 거고 더 좋은 방식이 있다면 다른 방식으로 하시면 될 것 같습니다! 초첨은 `round`수로 고정된 길이의 array로 생성 시에 상태를 부여하는 방식을 사용한다는 걸로 봐주면 될 것 같아요!

Reply:
from에 함수가 들어가는걸 몰랐네요... 안그래도 한번에 하고 싶었는데 반영하겠습니다!